buildstockbatch/base.py

```diff
@@ -215,11 +216,10 @@ def cleanup_sim_dir(sim_dir, dest_fs, simout_ts_dir, upgrade_id, building_id):
             units_dict = {}
             skiprows = []
 
-        schedules_filepath = ""
-        if os.path.isdir(os.path.join(sim_dir, "generated_files")):
-            for file in os.listdir(os.path.join(sim_dir, "generated_files")):
-                if re.match(r".*schedules.*\.csv", file):
-                    schedules_filepath = os.path.join(sim_dir, "generated_files", file)
+        schedules_filepaths = []
+        for file in os.listdir(output_dir):
```

Comment:
From reading the path in `sim_dir/generated_files/.*schedules.*.csv` to `sim_dir/run/in.schedules.*\.csv`, have  or do we need changed the place to save them?

Reply:
That's the path ResStock export the schedules in.
https://github.com/NREL/resstock/blob/17360a717583ab456812016625cc4357be9bd236/resources/hpxml-measures/HPXMLtoOpenStudio/measure.rb#L281